airbyte-webapp/src/views/layout/SideBar/components/NavDropdown.module.scss

```diff
@@ -0,0 +1,26 @@
+@use "scss/variables";
+@use "scss/colors";
+@forward "./NavItem.module.scss";
+
+.dropdownMenuButton {
```

Comment:
TIL that you can't use a functional component as the base for these dropdowns (so cannot use a `NavItem` and get rid of this, though I do reuse some of the classnames to reduce redundancy)

Reply:
I don't understand why we can't use functional components for `DropdownMenu` button... sunk 30 mins into it, but did not figure it out. This works fine too 🙂

Reply:
Thanks for the explanation! I knew something funky was going on with that render prop pattern, but I couldn't figure it out. That makes sense!